apis/apps/v1beta3/webhook_validation_test.go

```diff
@@ -63,6 +62,93 @@ var _ = Describe("EMQX Enterprise", func() {
 	})
 })
 
+var _ = Describe("EMQX Plugin", func() {
+	Context("Check EMQX emqx-management Plugin", func() {
```

Comment:
I think these two test cases can be merge in one `Context()`

Reply:
Done